tests/jobs/handleAutoResponse.test.ts

```
@@ -87,21 +87,20 @@ describe("handleAutoResponse", () => {
       where: eq(conversationMessages.id, message.id),
     });
     expect(updatedMessage?.metadata).toEqual(mockMetadata);
     expect(platformCustomer.upsertPlatformCustomer).toHaveBeenCalledWith({
       email: "customer@example.com",
-      mailboxId: mailbox.id,
       customerMetadata: mockMetadata.metadata,
     });
   });
 
   it("skips if conversation is spam", async () => {
     const { mailbox } = await mailboxFactory.create();
```

ESLint

[error] 'mailbox' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)